vagrant_repo/data_bags/cerner_splunk/multisiteUS.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "multisiteUS",
+  "sites" : ["cerner_splunk/cluster-vagrant-site1", "cerner_splunk/cluster-vagrant-site2"],
+  "settings": {
```

Comment:
Cluster came up clean.  I had also confirmed that all the buckets were replicated and that everything was green on the indexer clustering dashboard and that forwarders were forwarding and indexers were indexing. I wonder if I will see anything different if I bring site2 indexers first and then site1. Since that is what the docs says I think I should add it.